2020Main/src/main/java/frc/robot/OI.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot;
+
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj.buttons.JoystickButton;
+
+public class OI{
+
+    public static enum Axis{X, Y, Z, LeftY, RightY, THROTTLE, RightTab}; //Axis enum
+
+    public static Joystick driveStick;
+
+    public OI(){
+        driveStick = new Joystick(RobotMap.DRIVE_STICK);
+        assoc();
+    }
+
+    public static void getJoystickAxis(int joystickID,Axis axis){
```

Comment:
This and everything in it are for driving with an xbox controller, you can delete it